libraries/extensions/ros2-bridge/python/src/lib.rs

```diff
@@ -101,7 +101,7 @@ impl Ros2Node {
         message_type: String,
         qos: qos::Ros2QosPolicies,
     ) -> eyre::Result<Ros2Topic> {
-        let (namespace_name, message_name) = message_type.split_once("::").with_context(|| {
+        let (namespace_name, message_name) = message_type.split_once("/").with_context(|| {
```

Comment:
How about we allow both `::` and `/` as potential separators here? This way, we avoid breaking existing users.